src/components/logs-page/index.tsx

```diff
@@ -125,6 +126,28 @@ class LogsPage extends Component<PropsFromStore & BridgeApi & UtilsApi & WithTra
                                 onChange={updateBridgeConfig}
                             />
                         </div>
+                        <div className="col-12 col-sm-4 col-xxl-4">
+                            <label htmlFor="logs-limit" className="form-label">
+                                {t('logs_limit')}
+                            </label>
+                            <select
+                                id="logs-limit"
+                                className="form-select"
+                                onChange={(e) => {
+                                    const limit = parseInt(e.target.value);
+                                    store.setState({
+                                        logsLimit: limit,
+                                        logs: [...store.getState().logs.slice(-limit)],
+                                    });
+                                }}
+                            >
+                                {logLimits.map((limit) => (
+                                    <option key={limit} value={limit} selected={limit == store.getState().logsLimit}>
```

Comment:
i guess the `key` and `value` are unnecessary and can be removed?

i put them there, as i kinda copy-pasted `log-level` section ;)

Reply:
please refactor store.getState().logsLimit out of this loop, somewhere to the top of render function, example:

`const { logsLimit } = store.getState();`

Reply:
BTW, when I make a commit responding to a comment, I like to link the comment in the commit description (so that it gives more opportunity to discover why a particular change was made).

How do I link to _this_ comment? ;-)

My commit message:

```
perf(logs): extracted var to optimize a loop

https://github.com/nurikk/zigbee2mqtt-frontend/pull/1999/commits/9bca1ad6688ad003b85faf0de4acb4c9e467e075#r1574572268
```

The result:

```
22:46:37.622: [zigbee2mqtt-frontend] git -c credential.helper= -c core.quotepath=false -c log.showSignature=false add --ignore-errors -A -f -- src/components/logs-page/index.tsx
22:46:37.630: [zigbee2mqtt-frontend] git -c credential.helper= -c core.quotepath=false -c log.showSignature=false commit -F /private/var/folders/ws/nzgf19n976gfpx235sryx8lc0000gn/T/git-commit-msg-.txt --
⧗   input: perf(logs): extracted var to optimize a loop
https://github.com/nurikk/zigbee2mqtt-frontend/pull/1999/commits/9bca1ad6688ad003b85faf0de4acb4c9e467e075#r1574572268
✖   footer's lines must not be longer than 100 characters [footer-max-line-length]
✖   found 1 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
husky - commit-msg hook exited with code 1 (error)
```

Reply:
done https://github.com/nurikk/zigbee2mqtt-frontend/pull/1999/commits/94b0f5005e8dae66a0ac5d923315dc16441f3189